bson/bson_binary_vector_spec_test.go

```diff
@@ -82,21 +81,18 @@ func TestBsonBinaryVectorSpec(t *testing.T) {
 	})
 }
 
-func convertSlice[T int8 | float32 | byte](s []interface{}) []T {
+func convertSlice[T int8 | float32 | byte](t *testing.T, data []byte) []T {
```

Comment:
```suggestion
func convertSlice[T int8 | float32 | byte](t *testing.T, data []byte) []T {
	t.Helper()
```

Alternatively, this method could be re-written to return an error.